cmd/cli/app/upload.go

```diff
@@ -17,29 +17,22 @@ package app
 
 import (
 	"bytes"
-	"compress/gzip"
 	"context"
-	"crypto/sha256"
-	"encoding/hex"
 	"encoding/json"
-	"io"
 	"io/ioutil"
 	"net/http"
-	"os"
-	"strings"
+	"path/filepath"
 	"time"
 
 	"github.com/google/trillian"
-	"github.com/projectrekor/rekor/pkg"
 	"github.com/projectrekor/rekor/pkg/log"
+	"github.com/projectrekor/rekor/pkg/types"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"golang.org/x/crypto/openpgp"
-	"golang.org/x/crypto/openpgp/armor"
 )
 
 type RespStatusCode struct {
-	Code string `json:"file_recieved"`
+	Code string `json:"file_received"`
```

Comment:
Should this actually say "code"?

Reply:
Not sure its important given that its only used for unmarshalling the response. This should probably be pulled out into a common types/something.go file to maintain consistency b/t client and server. 

This would be addressed by #45 (if the code-gen approach was taken) or could be handled in another PR.